api/app/utils/testing.py

```diff
@@ -16,5 +27,4 @@ def setUp(self):
     def tearDown(self):
         db.session.remove()
         db.reflect()
-        db.engine.execute('DELETE FROM alembic_version;')
```

Comment:
Also, why did you remove this line? My understanding of this is that it "resets" the current migration?

Reply:
This was done in conjunction with switching to SQLLite for our tests. Previously we ran against the "main" postgres database, which already had migrations run against it and therefore had an alembic_version table. I believe now the tables are created directly in SQLLite with db.reflect() (@jaderabbit can correct me if I'm wrong?) - ie. no migrations therefore no alembic_version table.

Reply:
Okay cool, that makes sense :)